src/Sylius/Bundle/AdminBundle/Resources/config/routing/product.yml

```diff
@@ -79,3 +79,19 @@ sylius_admin_product_create_simple:
                     form: "@SyliusAdmin/Product/_form.html.twig"
                 route:
                     name: sylius_admin_product_create_simple
+
+sylius_admin_product_update:
```

Comment:
You should add `update` to except actions in `sylius_admin_product` route (line 6) or even change this line to `only: ['delete']`